src/ComfortPay/Message/AbstractSoapRequest.php

```diff
@@ -48,6 +88,10 @@ protected function getSoapClient()
             'keep_alive' => false,
             'soap_version' => SOAP_1_2,
             'cache_wsdl' => WSDL_CACHE_NONE,
+            'proxy_host' => $this->getParameter('proxyHost'),
```

Comment:
vieme toto upravit tak aby sa to tam vobec nenastavovalo ak tam tieto parametre niesu vyplnene? tzn. nech ten request vyzera ako doteraz ak sa nenastavi proxy